wordpress-mobile/WordPress-iOS
# Notices: Add optional Taptic feedback

This PR adds the ability to specify Taptic feedback to be triggered when Notices are displayed in the app. It also adds feedback for Media success and failure notices.

**To test:**

* In the media library, upload media successfully, and check that you feel haptic feedback when the notice appears.
* In the media library, begin some uploads, turn on airplane mode and quickly return to the app. Check that you feel haptic feedback when the notice appears. It seems like you have to be fast returning to the app – the system won't trigger the feedback if the app's not foreground & activate when the notices appears.
